### DISCUSSION TITLE
Anyone need a Cromenco kit?

### DISCUSSION THREAD

**Discussion by n9uxu** [thehighnibble/thehighnibble.github.io#44] [Cromemco Z-1 replica]
Greetings!  I've built and love my 8080 kit, and I ordered the Cromenco kit, but I have never opened it.  I'm beginning to think I never will as I have a huge pile of kits to build and vintage machines to restore, so if anyone wants one, I can always order another in the future.  $250 shipped CONUS which will save you a few bucks.  Paypal is fine.  Reach out if you are interested.  These kits are top notch.

**Comment by GrantMeStrength** [comment]
Is it still available?

**Reply by n9uxu** [reply]
Yep.

**Reply by GrantMeStrength** [reply]
Please ping me - I've room on my shelf for another 8080!

**Reply by n9uxu** [reply]
I've shot you an email to your gmail.

Dave

**Reply by wile1411** [reply]
what other kits are you working on? Interesting to know if there is any I don't know about.
